### PR TITLE
Feature/SG 293 - Add register loss

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: src/super_gradients/common/registry/README.md [hunk 1]
```diff
@@ -85,4 +85,45 @@ python main.py --config-name=my_recipe.yaml
...
+
+import torch
+from torch.nn.modules.loss import _Loss
+
+from super_gradients import Trainer, init_trainer
+from super_gradients.common.registry.registry import register_loss
+
+
+@register_loss("custom_rsquared_loss")
+class CustomRSquaredLoss(_Loss):
```
Comment (ofrimasad) [inline src/super_gradients/common/registry/README.md:103]: add hare a comment that says we are extending torch.nn.modules.loss._Loss

**Review by ofrimasad — COMMENTED** [review]
very nice. just one very minor comment

File: src/super_gradients/common/registry/README.md (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/common/registry/README.md]: Something like that ? https://github.com/Deci-AI/super-gradients/pull/406/commits/5d2c13c6057900a73f55aa287fc4e68b16dcc7f5

**Review by ofrimasad — APPROVED** [review]
LGTM
